docs/releases/3.1.txt

```diff
@@ -285,6 +285,17 @@ Requests and Responses
 * The new :meth:`.HttpRequest.accepts` method returns whether the request
   accepts the given MIME type according to the ``Accept`` HTTP header.
 
+Security
+~~~~~~~~
+
+* The :setting:`SECURE_REFERRER_POLICY` setting now defaults to
+  ``'same-origin'``. This means that
+  :class:`~django.middleware.security.SecurityMiddleware` will set the
+  :ref:`referrer-policy` header to ``same-origin`` on all responses that do not
+  already have it. This prevents the ``Referer`` header being sent to other
+  origins. To override this default, explicitly set
+  :setting:`SECURE_REFERRER_POLICY` to ``None``.
```

Comment:
We should add also a short note to the `Backwards incompatible changes in 3.1`, at least:
```
* :setting:`SECURE_REFERRER_POLICY` now defaults to ``'same-origin'``.
```

Reply:
Sure; I was following suite with #11621 and #11751 , both of which didn't appear in 3.0's "backwards inncompatible" changes. Want me to add for them too?

Reply:
They are in a separate subsection under "Backwards incompatible changes in 3.0", see [new-default-values-for-security-settings](https://docs.djangoproject.com/en/3.0/releases/3.0/#new-default-values-for-security-settings).

Reply:
Right you are, sorry I missed that, I copied the linking strategy :)